### ISSUE TITLE
Support for keyboard navigation in lua evaluation results

### ISSUE THREAD

**Issue by henrikx** [silverbulletmd/silverbullet#2150]
Originally posted here: https://community.silverbullet.md/t/keyboard-navigation-and-queries/4392

Navigating links with keyboard navigation can be done by moving the cursor with arrow keys to the link and pressing Ctrl+Enter to open it. However, when the link you want to press is inside the result of a SLIQ, how are you supposed to do this without switching to the mouse? If you try to move the cursor to the link, it switches to editing the query instead. The same goes for widgets etc. Just wondering if there are any plans to address this?

**Comment by henrikx** [comment]
Related: when editor-read-only mode is active, keyboard navigation is also hard. The cursor still exists as I can see the visible markdown moving as I move the cursor, but the cursor itself is not visible, which makes it hard to select links with the keyboard.
